### PR TITLE
1.0.2

### PR DESCRIPTION
**Pull request by augustak**
Added support for using APIs that use a KMS key for bucket encryption.
Added support for providing extra parameters to POST /predictions for APIs that require additional parameters in the JSON body.

### CODE REVIEW

File: las/client.py [hunk 1]
```diff
@@ -128,20 +128,26 @@ def put_document(document_path: str, content_type: str, presigned_url: str) -> s
         :type content_type: str
         :param presigned_url: Presigned upload url from :py:func:`~las.Client.post_documents`
         :type presigned_url: str
+        :param use_kms: Adds KMS header to the request to S3. Set to true if your API using KMS default encryption on
+        the data bucket
```
Comment (magnurud) [inline las/client.py]: Is this correct english? 

I would prefer: 
Set to true if your API **is** using KMS default encryption on the data bucket

File: tests/test_config.cfg [hunk 2]
```diff
@@ -1,5 +1,6 @@
 [default]
 endpoint = https://demo.api.lucidtech.ai/v1
-model_names = invoice, receipt
-document_paths = tests/example.jpeg, tests/example.jpeg
-document_mime_types = image/jpeg, image/jpeg
\ No newline at end of file
+model_names = invoice
```
Comment (magnurud) [inline tests/test_config.cfg]: Does this mean we will we no longer test for receipts?

File: las/client.py (hunk 1 above)
Comment (augustak) [reply las/client.py]: Corrected.

File: tests/test_config.cfg (hunk 2 above)
Comment (augustak) [reply tests/test_config.cfg]: Fixed

**Pull request merged**